lang/translation.json

```diff
@@ -376,51 +410,51 @@
       },
       {"tag" : "expl_overviewplot_daily",
         "en": "This graph shows the hourly average per group/station per hour of the day. It is calculated over all selected days, resulting in an average daily pattern of concentrations.",
-        "nl": "Deze grafiek laat het uurlijkse gemiddelde zien per groep/station per uur van de dag. Dit is uitgerekend over alle dagen die u heeft geselecteerd. Dit levert een gemiddelde dagelijkse gang van de concentraties op."
+        "nl": "Hier zie je de gemiddelde concentratie per uur van de dag, deze is gemiddeld over de gekozen periode."
       },
       {"tag" : "expl_overviewplot_expl_daily",
         "en": "If you click on a sensor, the average concentration per hour will be displayed. This allows you to see the average concentration at each hour of the day. The x-axis represents the hour of the day, and the y-axis represents the concentration of PM10 or PM2.5 in μg/m³.",
-        "nl": "Als je een sensor aanklikt, wordt de gemiddelde concentratie per uur getoond. Hierdoor zie je de gemiddelde concentratie op elk uur van de dag. De staafjes geven de spreidng weer, meer informatie is te vinden onder het 'Informatie'-tabblad. Op de x-as zie je het uur van de dag; op de y-as staat de concentratie PM10 of PM2,5 in ug/m3."
+        "nl": "Hier zie je de gemiddelde concentratie op elk uur van de dag. De staafjes geven de spreidng weer, meer informatie is te vinden onder het 'Informatie over de tool'-tabblad."
       },
       {"tag" : "word_indu_timeseries",
         "en": "Individual timeseries",
         "nl": "Individuele tijdreeks"
       },
       {"tag" : "expl_indu_timeplot",
         "en": "Here you can view the timeseries of the individual sensor.",
-        "nl": "Hier kunt u de tijdreeks van de individuele sensor bekijken."
+        "nl": "Op deze pagina kun je de metingen van losse sensoren bekijken uit de selectie die je net gemaakt hebt. Daarnaast kun je de maximale waarde instellen, meetwaardes boven deze waarde worden dan niet meer getoond. Vaak zijn de hoge metingen gemeten met een vieze of stukke sensor. Hierdoor niet realistisch voor de buitenlucht."
```

Comment:
'stukke sensor', is that correct Dutch? Last sentence is missing something

Reply:
Oke, I changed it to "kapotte"

Reply:
Senstence rewritten to have a better sentence